test/test_blocks.py

```diff
@@ -1043,6 +1043,90 @@ def generation():
         assert "At step 9" not in captured.out
 
 
+class TestProgressBar:
+    @pytest.mark.asyncio
+    async def test_progress_bar(self):
+        from tqdm import tqdm
```

Comment:
Let's add `tqdm` to `requirements.in`

Reply:
Nice test. I would also add a unit test to make sure the regular `gr.Progress()` with `track_tqdm=False` also works